groups/groups.yaml

```diff
@@ -1002,6 +1002,31 @@ groups:
       - ramamoorthypandiyaraja@gmail.com
       - spiffxp@google.com
 
+  - email-id: release-managers-admins@kubernetes.io
+    name: release-managers-admins
+    description: |-
+      Grants elevated privileges to SIG Release GCP projects.
+      Membership is restricted to Release Engineering subproject owners.
+    settings:
+      AllowExternalMembers: "true"
+      WhoCanJoin: "INVITED_CAN_JOIN"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+      WhoCanViewGroup: "ALL_MEMBERS_CAN_VIEW"
+      WhoCanDiscoverGroup: "ALL_IN_DOMAIN_CAN_DISCOVER"
+      WhoCanInvite: "ALL_MANAGERS_CAN_INVITE"
+      WhoCanAdd: "ALL_MANAGERS_CAN_ADD"
+      WhoCanApproveMembers: "ALL_MANAGERS_CAN_APPROVE"
+      WhoCanModifyMembers: "OWNERS_AND_MANAGERS"
+      WhoCanModerateMembers: "OWNERS_AND_MANAGERS"
+      ReconcileMembers: "false"
+    owners:
+      - caselim@gmail.com
+      - cmiles@pivotal.io
+      - saugustus@vmware.com
```

Comment:
why are we adding two accounts for Stephen and Tim?  This means twice as many credentials that could leak.  I'd like to make a policy that we explicitly NOT do this.